static/app/views/profiling/flamegraph.tsx

```diff
@@ -71,21 +55,20 @@ interface FlamegraphViewProps {
   location: Location;
   params: {
     eventId?: Trace['id'];
-    projectId?: Project['id'];
+    projectId?: Project['slug'];
   };
 }
 
 function FlamegraphView(props: FlamegraphViewProps): React.ReactElement {
   const api = useApi();
   const organization = useOrganization();
-  const location = useLocation();
 
   const [requestState, setRequestState] = useState<RequestState<ProfileGroup>>({
     type: 'initial',
   });
 
   const initialFlamegraphPreferencesState = useMemo((): DeepPartial<FlamegraphState> => {
-    return decodeFlamegraphStateFromQueryParams(location.query);
+    return decodeFlamegraphStateFromQueryParams(props.location.query);
```

Comment:
Why :D?

Reply:
Just trying to consolidate which `location` this component used. It was already using `props.location` below so no reason to also call `useLocation`.

Reply:
Ah yeah, good call